app/db/models/law_policy/family.py

```diff
@@ -208,7 +217,7 @@ class FamilyDocument(Base):
     )
     last_modified = sa.Column(
         sa.DateTime(timezone=True),
-        default=sa.func.now(),
+        server_default=sa.func.now(),
```

Comment:
Out of interest why the move to server_default?

Reply:
This is some sqlalchemy [wizardry](https://docs.sqlalchemy.org/en/20/core/defaults.html) :mage_woman: Basically, when you use server_default, the default value (in our case, the current timestamp) will be put into the DDL for the table and therefore inserted into the database if no value is provided. Default on the other hand is used when the value that's going to be inserted into that column is calculated by a function/procedure/trigger. Originally, I thought about using the triggers I've defined to update the values on INSERT and UPDATE, but realised it wasn't necessary to use a trigger for INSERT as I could simply define the default value in the DDL. The logic for when the UPDATE trigger is called though is a heck of a lot more complicated, so using default values isn't the best thing to do.